src/app/dashboard-widgets/add-codebase-widget/codebase-item/codebase-item.component.html

```diff
@@ -9,8 +9,7 @@
           <a href="{{codebase.attributes.url}}" target="_blank" class="f8-card-codebase-url truncate">{{codebase.name}}</a>
         </div>
         <div class="list-group-item-text f8-card-list-group-item-text">
-          codebase last updated 03/12/2018
-          <!-- replace with real code -->
+          last updated {{ lastUpdated | async | date:'short' }}
```

Comment:
Can this also handle the case when last updated is invalid or unknown to show a placeholder or empty?